tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+import os
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
```

Comment:
not sure why this is needed here (this module does not seem to use GPU)

Reply:
OK, I will remove it. I think it better to run the tests on CPU instead of GPU, especially on HPC's login node(it will use all the cards).